### PR TITLE
Physical_Engine: replace calls to GetShallowClone

### LINKED ISSUE #2555: Physical_Engine: Replace calls to Clone methods with calls to Base DeepClone or ShallowClone
#### Broken rules:
See https://github.com/BHoM/admin/issues/9

#### Suggestions to restore compliance:
https://github.com/BHoM/admin/issues/9#issuecomment-709032648

### PR DESCRIPTION
**Pull request by alelom**
### Issues addressed by this PR

Closes #2555

See BHoM/admin#9

### Test files

Just compliance, should mean no functional change

### CODE REVIEW

File: Physical_Engine/Modify/Transform.cs [hunk 1]
```diff
@@ -52,7 +53,7 @@ public static IFramingElement Transform(this IFramingElement framingElement, Tra
                 return null;
             }
 
-            IFramingElement result = framingElement.GetShallowClone() as IFramingElement;
+            IFramingElement result = framingElement.ShallowClone() as IFramingElement;
```
Comment (al-fisher) [inline Physical_Engine/Modify/Transform.cs:56]:
```suggestion
            IFramingElement result = framingElement.ShallowClone();
```

File: Physical_Engine/Modify/Transform.cs [hunk 2]
```diff
@@ -62,7 +63,7 @@ public static IFramingElement Transform(this IFramingElement framingElement, Tra
             {
                 if (framingElement.Location is Line)
                 {
-                    ConstantFramingProperty newProperty = property.GetShallowClone() as ConstantFramingProperty;
+                    ConstantFramingProperty newProperty = property.ShallowClone() as ConstantFramingProperty;
```
Comment (al-fisher) [inline Physical_Engine/Modify/Transform.cs:66]:
```suggestion
                    ConstantFramingProperty newProperty = property.ShallowClone();
```

File: Physical_Engine/Modify/Transform.cs [hunk 3]
```diff
@@ -90,7 +91,7 @@ public static IOpening Transform(this IOpening opening, TransformMatrix transfor
                 return null;
             }
 
-            IOpening result = opening.GetShallowClone() as IOpening;
+            IOpening result = opening.ShallowClone() as IOpening;
```
Comment (al-fisher) [inline Physical_Engine/Modify/Transform.cs:94]:
```suggestion
            IOpening result = opening.ShallowClone();
```

File: Physical_Engine/Modify/Transform.cs [hunk 4]
```diff
@@ -110,7 +111,7 @@ public static oM.Physical.Elements.ISurface Transform(this oM.Physical.Elements.
                 return null;
             }
 
-            oM.Physical.Elements.ISurface result = panel.GetShallowClone() as oM.Physical.Elements.ISurface;
+            oM.Physical.Elements.ISurface result = panel.ShallowClone() as oM.Physical.Elements.ISurface;
```
Comment (al-fisher) [inline Physical_Engine/Modify/Transform.cs:114]:
```suggestion
            oM.Physical.Elements.ISurface result = panel.ShallowClone();
```

**Review by al-fisher — COMMENTED** [review]
@alelom am I right is saying you can dispense with the `as "X"` now as well?

**Review by FraserGreenroyd — APPROVED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required
@BHoMBot check copyright-compliance
@BHoMBot check dataset-compliance
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer
@BHoMBot check versioning
@BHoMBot check dataset-compliance
@BHoMBot check copyright-compliance
@BHoMBot check ready-to-merge

**Pull request merged**